docs/introduction/console-commands-for-application-management-phing-targets.md

```diff
@@ -220,10 +220,24 @@ Great tool when you want to translate your application into another language.
 
 For more information about translations, see [the separate article](/docs/introduction/translations.md).
 
-## Customize Phing properties (paths etc.)
-You can customize any property defined in `build.xml` via configuration file `build/build.local.properties` (use `build/build.local.properties.dist` as a template).
+## Customization of Phing targets and properties
+<!-- TODO: Remove or update the note below -->
+*Note: This section describes customization of Phing targets in version of Shopsys Framework that hasn't been released yet.*
+*In `v7.2.0` and below, all Phing targets are defined in `build.xml` in the project repository only - you can directly modify them or delete them instead of overwriting them.*
+
+You can override and replace any Phing target or property defined in the `shopsys/framework` package by redefining it in your `build.xml` config.
+
+When you override a Phing target, the original is renamed to `shopsys_framework.TARGET_NAME`.
```

Comment:
I had no idea about this behavior :open_mouth: could you please add a note here that this is a behavior of phing itself and maybe link to docs?

Reply:
Is that even important for the users? I mean, I could link to https://www.phing.info/phing/guide/en/output/chunkhtml/ImportTask.html#idp4684, but would that be useful info for our users, or just a noise?

Reply:
Well, the first thing that occurred to me when I read this - "how is this implemented? How does it work?" and then I realized "ah, it is the Phing behavior, ok, then I do not need to know this"...This was from a point of view of the framework developer, so maybe from the point of a project developer, the information is not important as well. It is up to you :wink: